extra/modules/greenbids-real-time-data/src/test/java/org/prebid/server/hooks/modules/greenbids/real/time/data/util/TestBidRequestProvider.java

```diff
@@ -0,0 +1,91 @@
+package org.prebid.server.hooks.modules.greenbids.real.time.data.util;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.databind.node.TextNode;
+import com.iab.openrtb.request.Banner;
+import com.iab.openrtb.request.BidRequest;
+import com.iab.openrtb.request.Device;
+import com.iab.openrtb.request.Format;
+import com.iab.openrtb.request.Imp;
+import com.iab.openrtb.request.Site;
+import org.prebid.server.json.JacksonMapper;
+import org.prebid.server.json.ObjectMapperProvider;
+import org.prebid.server.proto.openrtb.ext.request.ExtRequest;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.function.UnaryOperator;
+
+public class TestBidRequestProvider {
+
+    public static final ObjectMapper mapper = ObjectMapperProvider.mapper();
+    public static final JacksonMapper jacksonMapper = new JacksonMapper(mapper);
+
+    private TestBidRequestProvider() { }
+
+    public static BidRequest givenBidRequest(
+            UnaryOperator<BidRequest.BidRequestBuilder> bidRequestCustomizer,
+            List<Imp> imps,
+            Device device,
+            ExtRequest extRequest) {
+
+        return bidRequestCustomizer.apply(BidRequest.builder()
+                .id("request")
+                .imp(imps)
+                .site(givenSite(site -> site))
+                .device(device)
+                .ext(extRequest)).build();
+    }
+
+    public static Site givenSite(UnaryOperator<Site.SiteBuilder> siteCustomizer) {
+        return siteCustomizer.apply(Site.builder().domain("www.leparisien.fr")).build();
+    }
+
+    public static ObjectNode givenImpExt() {
+        final ObjectNode bidderNode = jacksonMapper.mapper().createObjectNode();
+
+        final ObjectNode rubiconNode = jacksonMapper.mapper().createObjectNode();
```

Comment:
why not just `mapper.createObjectNode()`?

Reply:
fixed